.gitignore

```diff
@@ -3,6 +3,9 @@ compile_commands.json
 bundlex.sh
 bundlex.bat
 
+# Dir generated by tmp_dir ExUnit tag
+/tmp/
```

Comment:
Silly question - are you sure it is `/tmp/` and not `tmp/`?

Reply:
Updated